Automattic/portkey-android
# Add configure plugin 0.6.0

This PR partially reverts https://github.com/Automattic/stories-android/pull/674 by adding configure plugin version `0.6.0` which is hosted at S3. Fetchstyle plugin is still commented out because it's not yet published to S3.

### To test:
If `Installable Build` task is successful, it means the plugin works as expected.

### To test locally (not necessary):
1. Remove `gradle.properties` file
2. Run `./gradlew applyConfiguration`
3. Verify that `gradle.properties` file is there and contains the correct information
